device/arista/x86_64-arista_7800_sup/Arista-7808R3A-FM/asic.conf

```diff
@@ -1,2 +0,0 @@
-NUM_ASIC=12
```

Comment:
We need asic.conf  to see if the device is single or multi asic?
This PR https://github.com/Azure/sonic-buildimage/pull/9681 is just adding the pcie address to config_db. We still need `asic.conf`

Reply:
The `asic.conf` in use by sonic is left untouched, see https://github.com/Azure/sonic-buildimage/blob/master/device/arista/x86_64-arista_7800_sup/asic.conf

The 3 `asic.conf` removed by this PR are hwsku specific ones that have never been handled by the master code.
We used these internally to enable testing on different chassis sizes and kind of fabric cards.